README.md

```diff
@@ -239,12 +221,10 @@ for more details about how atomic swaps work.
 ### Dependencies
 
 1. [Go 1.17 or 1.18](https://golang.org/doc/install)
-2. [Node 16 or 17](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) is used to bundle resources for the browser interface. It's important to note that the DEX client has no external JavaScript dependencies. The client doesn't import any Node packages. We only use Node to lint and compile our own JavaScript and css resources.
-3. [dcrd](https://github.com/decred/dcrd) and [dcrwallet](https://github.com/decred/dcrwallet), installed from the [v1.7.x release binaries](https://github.com/decred/decred-release/releases), or built from the `release-v1.7` branches.
-4. [Bitcoin Core v0.21.x or v22.x](https://bitcoincore.org/en/download/) (bitcoind or bitcoin-qt) wallet. If you use v22, you must **not** use a "descriptor" wallet.
-5. At least 2 GB of available system memory.
+2. [Node 16 or 18](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) is used to bundle resources for the browser interface. It's important to note that the DEX client has no external JavaScript dependencies. The client doesn't import any Node packages. We only use Node to lint and compile our own JavaScript and css resources.
```

Comment:
No node 17? I've been using v14.20.0 for the longest...

Reply:
Then have an alternating maintenance schedule.  https://nodejs.org/en/about/releases/

![image](https://user-images.githubusercontent.com/9373513/182422138-6c6d2030-8ff2-4ae4-9e0a-2a264c150b26.png)

![image](https://user-images.githubusercontent.com/9373513/182422275-95496f89-8e78-4123-9055-14fd6f99aaea.png)

v17 reached EOL on 2022-06-01, leaving v16 as "Active LTS" and v18 as "Current".

Reply:
Oh wow, didn't know. Thanks!